components/slack/actions/create-reminder/create-reminder.js

```diff
@@ -0,0 +1,48 @@
+const slack = require("../../slack.app.js");
+const { WebClient } = require("@slack/web-api");
+module.exports = {
+  key: "slack-create-reminder",
+  name: "Create Reminder",
+  description: "Create a reminder",
+  version: "0.0.5",
```

Comment:
For the registry, set this to version `0.0.1` like with other actions. I realize this is hard to remember to do when you're iterating in your own account and have to publish new versions. We're planning to make this easier in the future.